### PR TITLE
Creating my BioDrop Profile

### PR DESCRIPTION
**Pull request by ALIPHATICHYD**
## Check List (Check all the applicable boxes) 

- [✅] My code follows the code style of this project.
- [✅] My change requires changes to the documentation.
- [✅] I have updated the documentation accordingly.
- [✅] All new and existing tests passed.
- [✅] This PR does not contain plagiarized content.
- [✅] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by ALIPHATICHYD — COMMENTED** [review]
Added A short Bio, and a link to my socials

File: data/ALIPHATICHYD.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "bio": "Software Engineer | Product Manager | Public Speaker"
+}
+"links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/ALIPHATICHYD",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
```
Comment (ThomasCode92) [inline data/ALIPHATICHYD.json:12]: I think you need to remove this trailing comma.

**Review by ALIPHATICHYD — COMMENTED** [review]
Removed the trailing comma.

File: data/ALIPHATICHYD.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Fuad ALIPHATIC",
+  "bio": "Software Engineer | Product Manager | Public Speaker"
```
Comment (sital002) [inline data/ALIPHATICHYD.json:3]:
```suggestion
  "bio": "Software Engineer | Product Manager | Public Speaker",
```
Please add a comma here

File: data/ALIPHATICHYD.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+}
+"links": [
+    {
+      "group": "Socials Primary",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/ALIPHATICHYD",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    }
+]
```
Comment (sital002) [inline data/ALIPHATICHYD.json:13]:
```suggestion
]
}
```

File: data/ALIPHATICHYD.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Fuad ALIPHATIC",
+  "bio": "Software Engineer | Product Manager | Public Speaker"
+}
```
Comment (sital002) [inline data/ALIPHATICHYD.json:4]:
```suggestion
```

**Pull request merged**